index.html

```diff
@@ -291,6 +312,59 @@
           </dt>
         </dl>
       </section>
+      <section>
+	<h2>
+	  Media Devices
+	</h2>
+	<p>
+	  The <code><dfn id='idl-def-PermissionName.camera'>camera</dfn></code>,
+	  <code><dfn id='idl-def-PermissionName.microphone'>microphone</dfn>
+	  </code>,
+	  and <code><dfn id='idl-def-PermissionName.speaker'>speaker</dfn>
+	  </code>
+	  permissions are associated with permission to use media
+	  devices as specified in [[mediacapture-main]].
+	</p>
+	<p>These three permissions have an associated PermissionDescriptor:
+	</p>
+	<dl class='idl' title='dictionary DevicePermissionDescriptor :
+	    PermissionDescriptor'>
+	  <dt>
+	    DOMString deviceId
```

Comment:
@alvestrand Are you making an attempt to model permission to obtain device labels and metadata via device enumeration, so that the user can make meaningful choices of device?